contactaccessrights/tests/phpunit/CRM/Contactaccessrights/BAO/RightsTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+use CRM_Contactaccessrights_BAO_Rights as ContactRights;
+use CRM_Contactaccessrights_Test_Fabricator_Rights as RightsFabricator;
+use CRM_HRCore_Test_Fabricator_OptionValue as OptionValueFabricator;
+/**
+ * Class CRM_Contactaccessrights_BAO_RightsTest
+ *
+ * @group headless
+ */
+class CRM_Contactaccessrights_BAO_RightsTest extends BaseHeadlessTest {
+
+  private $regionEntityType = 'hrjc_region';
+
+  private $locationEntityType = 'hrjc_location';
```

Comment:
Instead of hardcoding the types here, please use the values from the `CRM_Contactaccessrights_Utils_RightType_Location` and `CRM_Contactaccessrights_Utils_RightType_Region` classes